dark/index.html

```diff
@@ -190,9 +190,9 @@ <h1>
       </div>
     </center>
   </section>
-  
-  <section class="second-section">
-    <!-- About Section -->
+
+  <!-- <section class="second-section">

```

Comment:
Good job @Rishav1707
Can you remove the new section created and add all the content to that second-section.
And add that bg color to it